packages/corvid-e2e-tests/test/loginLogout.e2e.js

```diff
@@ -25,14 +24,9 @@ describe("login / logout", () => {
 
   describe("after login", () => {
     it("should clone without need to login again", async () => {
-      //login
-      const cliLoginCommandResult = await cliDriver.login();
-      const editorDriver = await connectToLocalEditor(
-        cliLoginCommandResult.editorDebugPort
-      );
-      const loginDriver = await editorDriver.waitForLogin();
-      await loginDriver.login(getCorvidTestUser());
-      await cliLoginCommandResult.waitForCommandToEnd();
+      // login via test cookies
+      const cliHackLoginCommand = await cliDriver.loginWithTestCookies();
```

Comment:
I don't think the word Hack is needed here :)
Perhaps `loginWithSled` or `e2eLogin` or just `cliLoginCommand`

Reply:
yeah... it's a leftover... i'll change it to something else